apps/i18n/common/en_us.json

```diff
@@ -19,6 +19,7 @@
   "clearPuzzleConfirmHeader": "Are you sure you want to start over?",
   "codeMode": "Code",
   "codeTooltip": "See generated JavaScript code.",
+  "completedWithoutRecommendedBlock": "Congratulations! You completed Puzzle {puzzleNumber}. (But you could use a different block for stronger code.)",
```

Comment:
Yeah, none of us are happy with that phrasing, but we also don't want to use the phrase "better code" cause it comes off as more judgmental. Suggestions welcome!

And this is for the case where they've completed the puzzle with the optimal number of lines, but also without using a required block. Ideally all of our levels will be designed such that that will never happen, but we've got it just in case.